app/src/main/java/ch/epfl/sdp/blindly/match/CardStackAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.sdp.blindly.match
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import ch.epfl.sdp.blindly.R
+
+class CardStackAdapter(
+        private var profiles: List<Profile> = emptyList()
+) : RecyclerView.Adapter<CardStackAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        return ViewHolder(inflater.inflate(R.layout.item_profile, parent, false))
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val profile = profiles[position]
+        val v = holder.itemView.findViewById(R.id.item_image) as ImageView
+        v.setImageResource(R.mipmap.blindly_launcher_foreground)
+        holder.name.text = "${profile.name}"
+        holder.age.text = profile.age.toString()
+        holder.itemView.setOnClickListener { v ->
+            Toast.makeText(v.context, profile.name, Toast.LENGTH_SHORT).show()
+        }
```

Comment:
Why does it display text ? It might be a good idea to add documentation to this function so that it is clearer for everyone.